examples/knowledge-base/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "dev": "next dev",
     "build": "next build",
-    "build:prod": "yarn generate-public-files && next build",
-    "start": "next dev",
-    "start:prod": "next start"
+    "build:prod": "yarn ts-node scripts/generate-public-files && next build",
+    "prebuild": "yarn ts-node scripts/generate-search-index",
```

Comment:
@denkristoffer I think we need to move this in build process, otherwise I've to specify it in Vercel  `yarn prebuild` as a step before build!? I don't think it's part or [yarn lifecycle](https://yarnpkg.com/advanced/lifecycle-scripts)

Reply:
Search didn't work on preview deployment

Reply:
It's a a part of the npm lifecycle, should work out of the box 🤔 Maybe it's npm only?

I actually had a general thought about these examples, should we move them to npm only and stop using yarn? There's no need to force that tool on users in my opinion.

Reply:
Actually it works with yarn as well for me.

Reply:
I think most likely something else is erroring on Vercel. The API route is not showing up in the dashboard 🤔